app/views/posts/show.html.erb

```diff
@@ -8,5 +8,15 @@
   <strong>Body:</strong>
   <%= @post.body %>
 </p>
+
+<div id="comments">
+  <% if @post.comments.present? %>
+    <h1>Comments</h1>
+  <% end %>
+  <%= render @post.comments %>
```

Comment:
Nit: The user won't see a difference, but if you're checking that comments exist anyway, you might as well move the `render @post.comments` line into that block so that Rails doesn't have to render an empty collection. It won't make a significant performance impact, but I feel like the structure makes more sense as

> if content is present:
> * header
> * content

versus

> if content is present:
> * header
>
> possible content?

```suggestion
  <% if @post.comments.present? %>
    <h1>Comments</h1>
    <%= render @post.comments %>
  <% end %>
```